web/src/components/profile/ProfileNoteForBusinessesFormedOutsideNavigator.tsx

```diff
@@ -0,0 +1,21 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { isFormedOutsideNavigator } from "@/lib/utils/helpers";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+import { ReactElement } from "react";
+
+interface Props {
+  business: Business | undefined;
+}
+
+export const ProfileNoteForBusinessFormedOutsideNavigator = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  return isFormedOutsideNavigator(props.business) ? (
```

Comment:
Since All we are doing here is returning an alert, is this component `ProfileNoteForBusinessFormedOutsideNavigator` necessary?

Reply:
It has a little bit of logic to see if it exists or not plus I kinda like the obfuscation with a name that says whats going on.